File: src/main/java/io/github/dqualizer/dqapi/repositories/dam/domainstory/ActivityRepository.java
package io.github.dqualizer.dqapi.repositories.dam.domainstory;

import io.github.dqualizer.dqlang.types.dam.domainstory.Activity;
import org.springframework.data.mongodb.repository.MongoRepository;

public interface ActivityRepository extends MongoRepository<Activity, String> {
}

File: src/main/kotlin/io/github/dqualizer/dqapi/repositories/dam/domainstory/ActivityRepository.kt
package io.github.dqualizer.dqapi.repositories.dam.domainstory

import io.github.dqualizer.dqlang.types.dam.domainstory.Activity
import org.springframework.data.mongodb.repository.MongoRepository
import org.springframework.stereotype.Repository

@Repository
interface ActivityRepository : MongoRepository<Activity, String>
